WordPress/src/main/res/values/strings.xml

```diff
@@ -263,6 +263,7 @@
     <!-- post view -->
     <string name="post_id">Post</string>
     <string name="upload_failed">Upload failed</string>
+    <string name="draft_uploaded">Draft uploaded</string>
```

Comment:
I wonder if we should reuse [`Draft saved online`](https://github.com/wordpress-mobile/WordPress-Android/blob/8728c6916ca9fd19f78e7fdd351ef0011b03c9bf/WordPress/src/main/res/values/strings.xml#L969) for this instead. We currently use it a little differently: in a snackbar when updating an _existing_ remote draft, but I wonder if it might make sense to unify the language here. I'd suggest getting some design input on this one.